examples/crud.rs

```diff
@@ -0,0 +1,56 @@
+extern crate iron;
+extern crate router;
+
+// To run, $ cargo run --example crud
+// To use: 
+// GET     http://127.0.0.1:3000/todos
+// POST    http://127.0.0.1:3000/todos
+// GET     http://127.0.0.1:3000/todos/:id
+// PATCH   http://127.0.0.1:3000/todos/:id
+// DELETE  http://127.0.0.1:3000/todos/:id
+
+use iron::{Iron, Request, Response, IronResult};
+use iron::status;
+use router::{Router};
+
+struct Controller;
```

Comment:
`mod controller { .. }` is more appropriate, with the now-methods being free functions instead.